### DISCUSSION TITLE
Add setting to set --wait parameter for Apex test run

### DISCUSSION THREAD

**Discussion by nufsty2** [forcedotcom/salesforcedx-vscode#5770] [Ideas]
**Is your feature request related to a problem? Please describe.**
With large Apex tests, the default --wait parameter is not long enough and the tests time out before coverage can be generated.

**Describe the solution you'd like**
Just a simple entry in the settings menu to set the --wait parameter value.

**Describe alternatives you've considered**
I can run "sfdx force:apex:test:report -i <ID> --wait <# minutes> -c" from the command line, but that doesn't generate a coverage report that is viewable in VSCode (to see exactly which lines are covered, etc). This is still viewable from the Developer Console, but who wants to use the Developer Console?

**Additional context**
Not a huge issue, would just be more complete and convenient to have the setting.

**Comment by Andrews-Consulting** [comment]
Are you actively building the code to address this issue?  If not, do you mind if I put some thought into it?

**Comment by diyer** [comment]
This has been resolved. https://github.com/forcedotcom/cli/issues/2108
